pennylane_lightning/lightning_qubit.py

```diff
@@ -93,7 +93,9 @@ def __init__(self, wires, *, shots=1000, analytic=True):
 
         if self.num_wires > self._MAX_WIRES:
             warnings.warn(
-                "The number of wires exceeds 16, reverting to NumPy-based evaluation.",
+                "The number of wires exceeds {}, reverting to NumPy-based evaluation.".format(
+                    self._MAX_WIRES
+                ),
```

Comment:
Could use an f string here
```suggestion
                f"The number of wires exceeds {self._MAX_WIRES}, reverting to NumPy-based evaluation."
```